src/components/Visualizations/Graph/MatchGraph.jsx

```diff
@@ -188,7 +188,7 @@ class PlayersGraph extends React.Component {
               {match.players.map((player) => {
                 const hero = heroes[player.hero_id] || {};
                 const playerColor = playerColors[player.player_slot];
-                const isSelected = (hoverHero === heroes[player.hero_id].localized_name);
+                const isSelected = (hoverHero === heroes[player.hero_id] && heroes[player.hero_id].localized_name);
```

Comment:
This line is a string comparison, I think you should move the object validation in front of the comparison.
Should look like this: `heroes[player.hero_id] && (hoverHero === heroes[player.hero_id].localized_name);`

Reply:
thanks. I changed it as you suggested.